src/modules/rover_mecanum/module.yaml

```diff
@@ -160,6 +176,21 @@ parameters:
         decimal: 2
         default: 0.5
 
+      RM_MAX_DECEL:
+        description:
+          short: Maximum deceleration
+          long: |
+            Maximum decelaration is used to limit the deceleration of the rover.
+            Set to -1 to disable, causing the rover to decelerate as fast as possible.
+            Caution: This disables the slow down effect in auto modes.
```

Comment:
You could be more clear what the "this" refers to - disabling or enabling this param.

Reply:
Agreed, updated in [b38caa5](https://github.com/PX4/PX4-Autopilot/pull/24036/commits/b38caa565970ecb26274f476046105fd070af6d9).